src/jit/compiler.h

```diff
@@ -487,6 +487,45 @@ class LclVarDsc
         assert(_lvRegNum == reg);
     }
 
+    const size_t lvArgStackSize() const
+    {
```

Comment:
You've place this in the middle of the lvRegNum/lvOtherReg definitions. Can you instead place it after lvSize()?